### PR TITLE
[EventHub]:Eventhub capture MSI support

### PR DESCRIPTION
**Pull request by schaudhari6254888**
## Checklist

- [ ] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** select appropriate branch. Cmdlets from Autorest.PowerShell should go to [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch. 
* **SHOULD** make the title of PR clear and informative, and in the present imperative tense. 
* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. Add changelog in description section if PR goes into [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** have approved design review for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) ([_Microsoft internal only_](../blob/main/CONTRIBUTING.md#onboarding)) with following situations
    * Create new module from scratch
    * Create new resource types which are not easy to conform to [Azure PowerShell Design Guidelines](../blob/main/documentation/development-docs/design-guidelines)
    * Create new resource type which name doesn't use module name as prefix
    * Have design question before implementation
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** introduce [breaking changes](../blob/main/documentation/breaking-changes/breaking-changes-definition.md) in Az minor release except preview version.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

**Review by damodaravadhani — COMMENTED** [review]
Remove this file

File: src/EventHub/EventHub.Autorest/11 [hunk 1]
```diff
@@ -0,0 +1 @@
+10
```
Comment (damodaravadhani) [inline src/EventHub/EventHub.Autorest/11:1]: Remove this file

File: src/EventHub/EventHub.Autorest/13 [hunk 2]
```diff
@@ -0,0 +1 @@
+12
```
Comment (damodaravadhani) [inline src/EventHub/EventHub.Autorest/13:1]: Remove this file

File: src/EventHub/EventHub.Autorest/README.md [hunk 3]
```diff
@@ -31,24 +31,24 @@ For information on how to develop for `Az.EventHub`, see [how-to.md](how-to.md).
...
-  - https://github.com/Azure/azure-rest-api-specs/blob/main/specification/eventhub/resource-manager/Microsoft.EventHub/preview/2022-10-01-preview/disasterRecoveryConfigs.json
-  - https://github.com/Azure/azure-rest-api-specs/blob/main/specification/eventhub/resource-manager/Microsoft.EventHub/preview/2022-10-01-preview/operations.json
-  - https://github.com/Azure/azure-rest-api-specs/blob/main/specification/eventhub/resource-manager/Microsoft.EventHub/preview/2022-10-01-preview/SchemaRegistry.json
-  - https://github.com/Azure/azure-rest-api-specs/blob/main/specification/eventhub/resource-manager/Microsoft.EventHub/preview/2022-10-01-preview/eventhubs.json
-  - https://github.com/Azure/azure-rest-api-specs/blob/main/specification/eventhub/resource-manager/Microsoft.EventHub/preview/2022-10-01-preview/ApplicationGroups.json
+  - https://github.com/dsouzaarun/azure-rest-api-specs/tree/ardsouza/main/EH/2023-01-01/specification/eventhub/resource-manager/Microsoft.EventHub/preview/2023-01-01-preview/AvailableClusterRegions-preview.json
```
Comment (damodaravadhani) [inline src/EventHub/EventHub.Autorest/README.md:40]: You wouldn't be allowed to merge this PR for any branch other then main in Azure fork. We should prioritize and merge that swagger

**Review by damodaravadhani — APPROVED** [review]

**Comment by schaudhari6254888** [comment]
Can someone review & merge this PR.

**Review by YanaXu — APPROVED** [review]

**Comment by YanaXu** [comment]
Hi @schaudhari6254888 , please leave a message for ChangeLog. Check examples in [ChangeLog.md](https://github.com/Azure/azure-powershell/blob/main/src/EventHub/EventHub/ChangeLog.md).

**Pull request merged**